modules/ROOT/pages/runtime-fabric-limitations.adoc

```diff
@@ -49,6 +49,7 @@ The following table lists known limitations for host operating systems in Runtim
 |===
 | Operating System | Limitation
 | RHEL 8 | If the host operating system is RHEL 8, do not set the optional environment variable `disable_selinux` to `false` in the `fabric.tf` script, or installation will fail.
+| RHEL 7 | If the host operating system is RHEL 7, and you've enable SELinux, third-party software running on the host may tweak policy settings from SELinux.
```

Comment:
```suggestion
| RHEL 7 | If the host operating system is RHEL 7, and you have enabled SELinux, third-party software running on the host may tweak policy settings from SELinux.
```

Reply:
Might check on using the word tweak because of negative drug-related connotations? But I won't die on this hill either way.